contracts/CptData.sol

```diff
@@ -24,9 +24,12 @@ contract CptData {
 
     uint constant private AUTHORITY_ISSUER_MAX_CPT_ID = 1999999;
 
-    uint authority_issuer_start_id = 100;
-
-    uint none_authority_issuer_start_id = 2000000;
+    // CPT ID has been categorized into 3 zones: 0 - 1000 are reserved for system CPTs,
+    //  1000-2000000 for Authority Issuer's CPTs, and the rest for common WeIdentiy DIDs.
+    uint constant private authority_issuer_start_id = 1000;
+    uint constant private none_authority_issuer_start_id = 2000000;
```

Comment:
Since you define a new constant variable for none_authority_issuer_start_id, I guess you can delete AUTHORITY_ISSUER_MAX_CPT_ID.

Reply:
correct, done.